plugins/serializers/graphite/README.md

```diff
@@ -64,4 +66,7 @@ cpu_usage_user;cpu=cpu-total;dc=us-east-1;host=tars 0.89 1455320690
 cpu_usage_idle;cpu=cpu-total;dc=us-east-1;host=tars 98.09 1455320690
 ```
 
+When the `graphite_tag_new_sanitize` option is enabled, the tags support the
+full range of characters defined by Graphite.
```

Comment:
could be worth expanding what the two sanitize settings are here

Reply:
Added a bunch of text that describes what each "mode" does.